src/main/webapp/js/user-page-loader.js

```diff
@@ -113,4 +113,4 @@ function buildUI() {
   showMessageFormIfViewingSelf();
```

Comment:
I think this needs to be updated to `showMessageFormIfLoggedIn` to match the renaming above.  Otherwise when I try to go to user-page.html the page just says "Loading..."  and I get a JavaScript error:

```
Uncaught ReferenceError: showMessageFormIfViewingSelf is not defined
    at buildUI (user-page-loader.js:113)
    at onload (user-page.html?user=test@example.com:26)
```

Reply:
Yup, it should've been changed. I fixed it